### PR TITLE
#2859 Rename "Cluster"XXX to "Service"XXX

### PR DESCRIPTION
**Pull request by chia7712**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #2859)
just rename files
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-it/src/test/scala/com/island/ohara/it/prometheus/TestPrometheus.scala [hunk 1]
```diff
@@ -43,14 +43,14 @@ class TestPrometheus extends IntegrationTest with Matchers {
   private[this] val nodes: Seq[Node] = EnvTestingUtils.sshNodes()
   private[this] var node: Node = _
   private[this] var nodeCollie: NodeCollie = _
-  private[this] var clusterCollie: ClusterCollie = _
+  private[this] var clusterCollie: ServiceCollie = _
```
Comment (saivirtue) [inline ohara-it/src/test/scala/com/island/ohara/it/prometheus/TestPrometheus.scala]: Maybe we will need another issue to rename all the "variables" from clusterXXX to serviceXXX ?

Comment (chia7712) [reply ohara-it/src/test/scala/com/island/ohara/it/prometheus/TestPrometheus.scala]: > Maybe we will need another issue to rename all the "variables" from clusterXXX to serviceXXX ?

let me address your comment in this one :)

**Comment by chia7712** [comment]
retry k8sConfigurator

**Comment by chia7712** [comment]
retry k8sConfigurator

**Pull request merged**